### ISSUE TITLE
Failed to run in docker

### ISSUE THREAD

**Issue by GeminiWind** [YellowLabTools/YellowLabTools#310]
**Build source**
[1.13.4](https://github.com/YellowLabTools/YellowLabTools/commit/8f5f9d6238cf2ab22d2c3bd6e32b62b18455225e)
**Step to reproduce**
- Build docker image from Dockerfile
- Running docker image
- Browse to local yellow lab and enter the site to analysties

**Expected**
It should analytics the site successfully

**Error**
The log show the error
```
Listening on port 8383
Launching test fnba7trbho on http://example.com
(node:1) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead.
fs.js:136
    throw new ERR_INVALID_CALLBACK();
    ^

TypeError [ERR_INVALID_CALLBACK]: Callback must be a function
    at makeCallback (fs.js:136:11)
    at Object.rmdir (fs.js:671:14)
    at Dir.rmdir (/app/ylt/node_modules/temporary/lib/dir.js:44:12)
    at /app/ylt/lib/screenshotHandler.js:160:33
    at FSReqWrap.args [as oncomplete] (fs.js:140:20)
```

**Comment by GeminiWind** [comment]
Pr to fix error in here #311

**Comment by ousamabenyounes** [comment]
Hello, I recommand you to use our new official V2 docker image
https://github.com/ousamabenyounes/docker-yellowlabtools

`docker run --privileged -p 8383:8383 ousamabenyounes/yellowlabtools 
`
